pkg/models/blog.go

```diff
@@ -33,19 +33,18 @@ func logFatal(err error) {
 }
 
 func (b *BlogStore) GetBlog(userID int) []Blog {
-	rows, err := b.DB.Query("select blog_id,pets.user_id,name,created_time,content from blog,pets where pets.user_id  = $1 order by created_time desc ", userID)
+	rows, err := b.DB.Query("select blog_id, content, created_time, name from blog,pets where blog.user_id =$1 and pets.user_id=blog.user_id order by created_time desc;", userID)
 	if err != nil {
 		logFatal(err)
```

Comment:
If err != nil then return an error and handle it in contoller. 
No fatal errors in models at all! Remove logFatal(err) func.
By the way, db query error is not fatal, it is a simple error.
Handle erros with logger package in controllers. Look at already written code.